### PR TITLE
chore(puppeteer): update to support `puppeteer@latest`

### PR DESCRIPTION
**Pull request by michael-siek**
See: #105 

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**

- [x] Code is reviewed for security

### CODE REVIEW

File: packages/puppeteer/package.json
Comment (stephenmathieson) [inline packages/puppeteer/package.json:37]: Since there is a `package-lock.json` file here, using `latest` doesn't really mean "latest". I think we ought to pin v5 here instead.

File: packages/puppeteer/package.json
Comment (stephenmathieson) [inline packages/puppeteer/package.json:48]: This isn't safe. If Puppeteer v6 removes functionality we rely on, we'll be telling our users that this package works with v6 while it actually does not.

**Review by stephenmathieson — CHANGES_REQUESTED** [review]

File: packages/puppeteer/package.json
Comment (michael-siek) [reply packages/puppeteer/package.json:48]: I could do something like 
```json
	"puppeteer": ">=1.10.0 < 6"
```

Comment (stephenmathieson) [reply packages/puppeteer/package.json:48]: That seems reasonable to me

**Review by stephenmathieson — COMMENTED** [review]
LGTM! Should get an approval from a maintainer tho 😉

**Review by stephenmathieson — APPROVED** [review]

**Pull request merged**
